ebu_tt_live/node/switcher.py

```diff
@@ -0,0 +1,6 @@
+
+from .base import AbstractCombinedNode
+
+
+class SwitcherNode(AbstractCombinedNode):
+    pass
```

Comment:
Well I followed the family tree described in the spec. I do not know if this class would ever be anything but the handover subclasses it.